src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
Let it be `calculateArea()`

Reply:
ok, but it mark like "Bad example" in our checklist